packages/bpk-component-text/package.json

```diff
@@ -15,6 +15,7 @@
   "dependencies": {
     "bpk-mixins": "^17.0.15",
     "bpk-react-utils": "^2.3.8",
+    "bpk-component-theme-switcher": "^0.0.0",
```

Comment:
Why is the text component getting a dependency on theme switcher?

Reply:
Good catch, missed that

Reply:
that's a very good question!